.github/workflows/actionlint.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - id: get_yamls
         run: |
           yamls=$(find . -name "*.yaml")
-          echo "::set-output name=files::${yamls}"
+          echo "files=${yamls}" >> $GITHUB_OUTPUT
```

Comment:
I think we need to quote the yamls, since they'll be newline-delimited.

Reply:
thanks!